pkg/sshd/server_test.go

```diff
@@ -8,6 +10,7 @@ import (
 	"github.com/deis/builder/pkg/controller"
 
 	"github.com/Masterminds/cookoo"
+	"github.com/arschles/assert"
```

Comment:
k my only concern is the day you leave =P and we are dependent on a package we dont control